src/ResourceMover/Az.ResourceMover.psd1

```diff
@@ -0,0 +1,24 @@
+@{
+  GUID = '97d87543-8eef-4574-a70d-7317ec4abe54'
+  RootModule = './Az.ResourceMover.psm1'
+  ModuleVersion = '1.0.0'
```

Comment:
The initial version should be 0.1.0

Reply:
Updated the version.

Reply:
I cannot see the change. Did you push your commit?

Reply:
Not sure why it is not showing here. But I am able to see the commit and in the individual file, I am able to see the change.